charts/internal/cilium/charts/agent/templates/daemonset.yaml

```diff
@@ -183,29 +187,14 @@ spec:
             type: spc_t
           capabilities:
             add:
-              # Use to set socket permission
               - CHOWN
-              # Used to terminate envoy child process
               - KILL
-              # Used since cilium modifies routing tables, etc...
               - NET_ADMIN
-              # Used since cilium creates raw sockets, etc...
               - NET_RAW
-              # Used since cilium monitor uses mmap
               - IPC_LOCK
-              # Used in iptables. Consider removing once we are iptables-free
               - SYS_MODULE
-              # We need it for now but might not need it for >= 5.11 specially
-              # for the 'SYS_RESOURCE'.
-              # In >= 5.8 there's already BPF and PERMON capabilities
               - SYS_ADMIN
-              # Could be an alternative for the SYS_ADMIN for the RLIMIT_NPROC
               - SYS_RESOURCE
-              # Both PERFMON and BPF requires kernel 5.8, container runtime
-              # cri-o >= v1.22.0 or containerd >= v1.5.0.
-              # If available, SYS_ADMIN can be removed.
-              #- PERFMON
-              #- BPF
```

Comment:
This is different from how cilium does it. Now, this is specified in `values.yaml` in upstream cilium. Please adapt accordingly.